frontend/public/components/utils/events-stream.tsx

```diff
@@ -0,0 +1,372 @@
+import * as React from 'react';
```

Comment:
Nit: please use the singular form, `event-stream.tsx`.

Reply:
using singular for now